Makefile

```diff
@@ -0,0 +1,21 @@
+.PHONY: help
+help: ## Display this help.
```

Comment:
We probably need a better description when to run this test: before, after, or instead of the deployment via `helm install`

Reply:
Added some words in CONTRIBUTING.md + added small improvement on test task documentation